src/WebJobs.Extensions.DurableTask/EntityScheduler/Entity.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// Statically accessible context for entity operations.
+    /// </summary>
+    public static class Entity
+    {
+        private static readonly AsyncLocal<IDurableEntityContext> EntityContext
+            = new AsyncLocal<IDurableEntityContext>();
+
+        /// <summary>
+        /// The context of the currently executing entity.
+        /// </summary>
+        public static IDurableEntityContext Context => EntityContext.Value;
```

Comment:
Other APIs in .NET name static properties like this `Current`. For example, `Thread.CurrentThread`, `ClaimsPrincipal.Current`, and WCF's `OperationContext.Current`, etc. I suggest we follow this convention, and name this class `EntityContext` with a property named `Current` - e.g. `EntityContext.Current`.

Reply:
I am fine with `Current`. This would then be called `Entity.Current` which I think is o.k.

BTW, I prefer to keep the static class name as `Entity` as opposed to changing it to `EntityContext` because I think length matters here , see next comment.